meson.build

```diff
@@ -105,14 +105,7 @@ math = cc.find_library('m')
 rt = cc.find_library('rt')
 
 if not get_option('xdg-foreign').disabled()
```

Comment:
We can probably remove this option, since `xdg-foreign` doesn't have any external dep anymore.